config/nunjucks/filters.js

```diff
@@ -114,6 +114,7 @@ function formatDateRange(value, delimiter = 'to') {
  * @example {{ "2019-02-21" | formatDateRange("DD/MM/YY") }}
  */
 function formatDateRangeAsRelativeWeek(value) {
+  const i18n = require('../i18n')
```

Comment:
This is obviously awful - just a stopgap to prompt discussion of how to prevent the circular dependency between config/118n and nunkucks/filters

(I did point out before that I didn't like the i18n.t lurking in here)

As no lang information is actually passed into the function, the i18n value returned would always be the default language. Is there any reason not to just grab the value for the string from the locales file directly? Or perhaps better, a specific config/locales file for the filters>

Or should we pass the i18n method in to either initialise the filters or return an instance?

...

Also should we move the contents of nunjucks/filters to common/lib/filters and then load that from the nunucks and the i18n modules